src/js/main.js

```diff
@@ -34,6 +46,60 @@ class Site {
       $(this).html(string);
     });
   }
+
+  dotSize() {
```

Comment:
I don't know how we do it with the new js markup `class` but I think the dot functions should be separated somehow like we used to do with the old markup.